server/lib/insightProviders/pageInsightProviders/googlePageSpeedInsights.js

```diff
@@ -19,11 +20,11 @@ module.exports = function googlePageSpeedInsights(url) {
 			link: `https://developers.google.com/speed/pagespeed/insights/?url=${url}&tab=mobile`
 		},{
 			name: 'WeightOfResources',
-			value: parseInt(results.pageStats.htmlResponseBytes, 10) + 
-					parseInt(results.pageStats.cssResponseBytes, 10) + 
-					parseInt(results.pageStats.imageResponseBytes, 10) + 
-					parseInt(results.pageStats.javascriptResponseBytes, 10) + 
-					parseInt(results.pageStats.otherResponseBytes, 10),
+			value: parseInt(results.pageStats.htmlResponseBytes, 10) || 0 + 
```

Comment:
Do you know which URL caused the error to surface? It would be great to see what Google's PageSpeed Insights returns in those cases.